scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -1564,13 +1575,16 @@ function Reset-SystemPath {
     Write-Verbose 'Removed IoT Edge directories from system PATH'
 }
 
+function Test-VcRuntimePresent {
+    return Test-Path 'C:\Windows\System32\vcruntime140.dll'
+}
```

Comment:
Missing empty line between the end of this function and the start of the next.

Reply:
fixed